docs/getting-started-developers.md

```diff
@@ -2,31 +2,31 @@
 
 ## Prerequisites
 
-[Git](https://git-scm.com/downloads), [Yarn](https://yarnpkg.com/en/docs/install), [Node.js](https://nodejs.org/en/download/), [Gulp](https://gulpjs.com/), [.NET](https://dotnet.microsoft.com/download/dotnet-core/2.2) and [VSCode](https://code.visualstudio.com/) must be installed prior to running the installation or build scripts.
-
-Run the command
-
-```
-npm config set scripts-prepend-node-path true
-```
-
-to tell VSCode which Node version to run during the extension compilation (otherwise you'll get an error during the build process).
+1. Install [Node.js](https://nodejs.org/en/download/)
+2. Install [Gulp](https://gulpjs.com/)
+3. Install [Git](https://git-scm.com/downloads)
+4. Install [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/2.2)
+5. Install [Visual Studio Code](https://code.visualstudio.com/)
+6. Install [Yarn](https://yarnpkg.com/en/docs/install). You will need to finish installing Node.js before you install Yarn.
+7. Run the command `npm config set scripts-prepend-node-path true`. This tells VSCode which Node version to run during the extension compilation (otherwise you'll get an error during the build process).
 
 _Note: If using Windows, use Git Bash_.
 
-## Quick Start
+## Running the Extension
 
-To get started, the first step is to clone this repository. To copy dependent code from submodule, install dependencies, compile the client, and compile the
-extension, run:
+1. Clone the repository with `git clone https://github.com/microsoft/WebTemplateStudio.git`
```

Comment:
Maybe put the ssh link too haha, if you're putting the https link might as well right?

Reply:
Sure

Reply:
I think it should be okay to assume that they can set up SSH keys?

Reply:
Yeah since this is getting started for developers. Can safely assume that